### PR TITLE
fix: sort button semi-hidden on mobile

### LINKED ISSUE #7154: Nova wallet: A button in profile is semi-hidden
## What happened
A button in the profile on the mobile app - Nova Wallet is semi-hidden. Open menu, go to profile, click on View profile and you will get here:
Look at the picture: Button Sort is cut off. 
IMG_6879

Please move this button a bit.

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7154
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image
image

### <samp>🤖 Generated by Copilot at c8d2144</samp>

Improved the responsiveness and layout of the `OrderByDropdown` and `ChainDropdown` components in the profile page. Added a function to position the `OrderByDropdown` dynamically and adjusted the spacing and wrapping of the components.

### <samp>🤖 Generated by Copilot at c8d2144</samp>

> _To make the profile detail look nice_
> _We tweaked the spacing and the size_
> _We added `calcDropdownPosition`_
> _To avoid any collision_
> _And used `is-flex-wrap-wrap` to wrap the `ChainDropdown` and `OrderByDropdown` twice_

### CODE REVIEW

File: components/profile/OrderByDropdown.vue [hunk 1]
```diff
@@ -127,6 +142,7 @@ onMounted(() => {
       selectedSort.value = [sort] as string[]
     }
   }
+  calcDropdownPosition()
```
Comment (Jarsen136) [inline components/profile/OrderByDropdown.vue:145]: What if the screen size changes? consider using `useEventListener(window, 'resize', fn)`

**Review by Jarsen136 — COMMENTED** [review]
TBH I'm not a fan of calculating the position for every component that uses tooltip component. It's better to put the calculation logic inside the Tooltip component. Or we could found out another component lib that providing such functionality.

**Comment by floyd-li** [comment]
> TBH I'm not a fan of calculating the position for every component that uses tooltip component. It's better to put the calculation logic inside the Tooltip component. Or we could find another component lib that provides such functionality.

yes i've put the logic into the `NeoDropdown` component

File: components/profile/OrderByDropdown.vue [hunk 2]
```diff
@@ -7,10 +7,11 @@
       multiple
       :mobile-modal="false"
       aria-role="list"
-      position="bottom-left"
+      position="bottom-auto"
       @change="onChange">
       <template #trigger="{ active }">
         <NeoButton
+          ref="buttonRef"
```
Comment (Jarsen136) [inline components/profile/OrderByDropdown.vue:14]:
```suggestion
```

**Review by Jarsen136 — APPROVED** [review]
good job

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 4.23 USD/DOT ~ 11.82 $DOT
🧗 [13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A](https://polkadot.subscan.io/account/13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)
🔗 [0x4b6e12426d45504571f642d1c1693908a7cf2a6533840e6877b9577d5802f980](https://polkadot.subscan.io/extrinsic/0x4b6e12426d45504571f642d1c1693908a7cf2a6533840e6877b9577d5802f980)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**